### DISCUSSION TITLE
TCAS

### DISCUSSION THREAD

**Discussion by IlanMoshe** [flybywiresim/aircraft#5179] [Help] [answered]
Hello the community,

I'm using the dev version.
In the last update it was noticed that SimConnect injected traffic (e.g. VATSIM, IVAO) in the TCAS. 
I have to set up something in the FS2020 settings ?

Thank you,

**Comment by hotshotp** [comment, marked as answer]
tcas shows up right now in the nd and a more comprehensive tcas system is being worked on
